cnd/src/config/file.rs

```diff
@@ -246,6 +255,15 @@ node_url = "http://localhost:8545/"
                     node_url: "http://localhost:8545".parse().unwrap(),
                 }),
             }),
+            lnd: Some(Lnd {
+                http_rpc_socket: Socket {
+                    address: IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)),
+                    port: 443,
+                },
+                macaroon: Some(PathBuf::from(
```

Comment:
Isn't the path to the tls cert needed too?

Reply:
Yes! Good point! We need the TLS certificate too :)

Reply:
Please see https://github.com/comit-network/comit-rs/pull/2131 (final patch) which implements functions to get the TLS cert as well as the macaroon.